src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.ts

```diff
@@ -31,12 +31,20 @@ export class EditSpaceDescriptionWidgetComponent implements OnInit {
     private userService: UserService,
     private broadcaster: Broadcaster,
     private spaceService: SpaceService,
-    private spaceNamespaceService: SpaceNamespaceService
+    private spaceNamespaceService: SpaceNamespaceService,
+    private collaboratorService: CollaboratorService
   ) {
-    spaces.current.subscribe(val => {
-      this.space = val;
-      console.log('newspace', val);
-    });
+    spaces.current
+      .subscribe(space => {
+        this.space = space;
+        if (space) {
+          this.collaboratorCount = this.collaboratorService.getInitialBySpaceId(space.id).map(c => c.length);
+          this.spaceOwner = this.userService.getUserByUserId(space.relationships['owned-by'].data.id).map(u => u.attributes.username);
+        } else {
+          this.collaboratorCount = Observable.empty();
```

Comment:
I would be inclined to move these into ngOnInit instead of being in the constructor. Is there code style on this? 

If space is null or undefined, the empty Observables will persist for the component. Can an error notification be triggered as well? A warning to the user would be good as well.